BrainPortal/public/stylesheets/new_account_demands.css

```diff
@@ -0,0 +1,183 @@
+
+body {
```

Comment:
Why is there a full stylesheet here? This is not needed if the registering mechanism is integrated within CBRAIN...